modules/odc-deploying-java-applications.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-deploying-java-applications.adoc
+
+[id="odc-deploying-java-applications_{context}"]
+= Upload JAR files for easy deployment of Java applications
+
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+
+* Navigate to *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file or drag and drop a JAR file to deploy your application.
+
+* Navigate to the *Topology* view and use the *Upload JAR file* option or drag and drop a JAR file to deploy your application.
+
+* Navigate to the Context Menu in the *Topology* view and use the *Upload JAR file* option to upload your JAR file to deploy your application.
+
+Use the following instructions to upload a JAR file in the *Topology* view to deploy a Java application.
+
+.Procedure
+
+. In the *Topology* view, right-click anywhere in the *Topology* view to see the *Add to Project* menu.
+
+. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form . Alternatively, you can drag and drop the JAR file in the *Topology* view.
+
+. In the *JAR file* field, browse for the required JAR file on your local machine and upload it. Alternatively, you can drag and drop the JAR file on the field. A toast alert is displayed at the top right if an incompatible file type is dragged and dropped on the *Topology* view. On the other hand, a field error is displayed if an incompatible file type is dropped on the field in the upload form.
+
+. You can further specify optional Java commands to customize your deployed application. The Runtime Icon and Builder Image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+
+. In the optional *Application Name* field, enter a unique name for your application for the resource labelling. 
+
+. In the *Name* field, enter a unique component name to name the associated resources.
+
+. In the *Resources* field, choose the resource type for your application.
+
+. In the *Advanced options*, click on *Create a Route to the Application* to configure a public URL for your deployed application.
+
+. Click *Create* to deploy the application. If there are no errors during the form submission the user will see a toast notification on the top right notifying that the JAR file is being uploaded and will take a while. The toast notification also includes a link to view the build logs.
+
++
+[NOTE]
+====
+If the user attempts to close the browser tab, while the build is running a web alert would be displayed asking the user if they really want to leave the page.
```

Comment:
If I'm understanding this sentence correctly, I believe the comma would go after `while the build is running` So it'd be:

`If the user attempts to close the browser tab while the build is running, a web alert would be displayed asking the user if they actually want to leave the page.`

Reply:
Thanks! Fixed :rocket: